### PR TITLE
Add support for workload show

### PR DESCRIPTION
**Pull request by kaccardi**
Modify controller and ciao-cli to support a "workload show" command. This command will display a workload definition and config in yaml format.

Fixes: #1012 #1013 #1134

### CODE REVIEW

File: ciao-cli/workload.go [hunk 1]
```diff
@@ -256,6 +257,52 @@ func optToReq(opt workloadOptions, req *types.Workload) error {
...
+	}
+
+	b, err := yaml.Marshal(opt)
+	if err != nil {
+		fatalf(err.Error())
+	}
+
+	fmt.Println(string(b))
+	fmt.Println(w.Config)
+
```
Comment (markdryan) [inline ciao-cli/workload.go]: There's an extra new line here.

**Comment by markdryan** [comment]
@kaccardi Works well for me.  One question.  Can you use this command to see other people's workloads?

**Comment by kaccardi** [comment]
@markdryan no, it will only show the workload associated with a particular tenant for non-privileged and for privileged it will show only public workloads for now. the next phase of this project is to allow privileged users to operate on tenant private workloads (show/delete/list). #1136 is the issue associated with this task.

**Comment by markdryan** [comment]
@kaccardi in which case it's good for me.   My only comment is an extra newline.  Feel free to ignore.

File: ciao-cli/workload.go (hunk 1 above)
Comment (kaccardi) [inline ciao-cli/workload.go]: @markdryan thanks for catching that, I pushed an updated commit without the extraneous newline.

**Pull request merged**